ghost/collections/src/CollectionsService.ts

```diff
@@ -144,6 +144,16 @@ export class CollectionsService {
                 sort_order: index
             }))
         };
+        if (collection.slug === 'latest') {
+            const allPosts = await this.postsRepository.getAll({ // TODO Support getting just ids
```

Comment:
We can drop down to a pure knex query straight away here to avoid the bookshelf pitfall again, right?

Reply:
Needs to be inside the `PostRepository` implementation, but yeah